pages/about/staff/_slug.vue

```diff
@@ -4,23 +4,25 @@
         <!-- no search on this page -->
         <nav-breadcrumb :title="page.entry.title" />
 
-        <block-staff-detail
-            class="staff-detail-block"
-            :image="parsedImage"
-            :staff-name="parsedStaffName"
-            :alternative-full-name="parsedAlternativeFullName"
-            :language="parsedLanguage"
-            :job-title="page.entry.jobTitle"
-            :departments="page.entry.departments"
-            :locations="page.entry.locations"
-            :pronouns="page.entry.pronouns"
-            :email="page.entry.email"
-            :phone="page.entry.phone"
-            :consultation="page.entry.consultation"
-            :topics="page.entry.topics"
-            :academic-departments="page.entry.academicDepartments"
-            :biography="page.entry.biography"
-        />
+        <section-wrapper>
+            <block-staff-detail
+                class="staff-detail-block"
+                :image="parsedImage"
+                :staff-name="parsedStaffName"
+                :alternative-full-name="parsedAlternativeFullName"
+                :language="parsedLanguage"
+                :job-title="page.entry.jobTitle"
+                :departments="page.entry.departments"
+                :locations="page.entry.locations"
+                :pronouns="page.entry.pronouns"
+                :email="page.entry.email"
+                :phone="page.entry.phone"
+                :consultation="page.entry.consultation"
+                :topics="page.entry.topics"
+                :academic-departments="page.entry.academicDepartments"
+                :biography="page.entry.biography"
+            />
+        </section-wrapper>
 
```

Comment:
we want to have a divider here after block-staff-detail if there are publications or orcid